shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -155,14 +172,7 @@ public InitResult call() {
 
             // Prefetch the default font manager as soon as possible on a background thread.
             // It helps to reduce time cost of engine setup that blocks the platform thread.
-            Executors.newSingleThreadExecutor()
-                .execute(
-                    new Runnable() {
-                      @Override
-                      public void run() {
-                        flutterJNI.prefetchDefaultFontManager();
-                      }
-                    });
+            executorService.execute(() -> flutterJNI.prefetchDefaultFontManager());
```

Comment:
The default `executorService` is `Executors.newSingleThreadExecutor()` which execute task sequentially, so will this change slow down the engine startup?

Reply:
Good catch. Changing this to cached thread pool and adding a test.